semcore/dropdown/src/index.d.ts

```diff
@@ -31,14 +31,38 @@ export type DropdownHandlers = PopperHandlers & {};
 
 export type DropdownTriggerProps = PopperTriggerProps;
 
+/**
+ * DropdownPopper must have an accessible name (aria-dialog-name).
+ * It should describe popper content.
+ */
+type DropdownPopperAriaProps = Intergalactic.RequireAtLeastOne<{
+  'aria-label'?: string;
+  'aria-labelledby'?: string;
+  title?: string;
+}>;
+
+export type DropdownPopperProps = PopperPopperProps &
+  DropdownPopperAriaProps & {
+    /**
+     * Popper in Dropdown should have role `dialog`.
+     * @default 'dialog'
+     */
+    role?: 'dialog';
```

Comment:
But it should not be dialog in some cases like tooltips, isn't it?
if so, you need to enforce it in Tooltips&co. in the same PR

Reply:
`Tooltip` use `Popper` as base, not Dropdown. So, no, Dropdown has only `dialog` right now.